src/Containers/CadView.jsx

```diff
@@ -215,6 +215,7 @@ export default function CadView({
     createPlaceMark({
       context: viewer.context,
       oppositeObjects: [tmpModelRef],
+      postProcessor: viewer.postProcessor,
```

Comment:
Yep, I wanted to know how the post processor of IFC viewer works?
Anyway, everything works well on my side. :)
Thank you.